pkg/openstack-cinder/config/cloudinfo.go

```diff
@@ -81,6 +82,15 @@ func getCloudInfo() (*CloudInfo, error) {
 	ua := gophercloud.UserAgent{}
 	ua.Prepend(fmt.Sprintf("csi-operator/%s", version.Get().GitCommit))
 
+	// our assets mount the CA file at a known path and we don't want to rely on other things
+	// setting the 'cafile' value in our clouds.yaml file to the same path, so we explicitly
+	// override this if a CA file is present
+	if _, err := os.Stat("/etc/kubernetes/static-pod-resources/configmaps/cloud-config"); err == nil {
```

Comment:
it's worth creating a constant for the path `/etc/kubernetes/static-pod-resources/configmaps/cloud-config`

Reply:
I'll move this if I need to respin.